README.md

```diff
@@ -76,15 +78,15 @@ If you would like to be notified of future conferences that would be added to th
 | [PyCon India](https://in.pycon.org/2016/) | 24-25 Sept | Jawahar Lal Nehru University, New Delhi | The premier conference in India on using and developing the Python programming language. |
 | [Meta Refresh](https://metarefresh.in/2016/) | 17 Sept | MLR Convention Centre, Bengaluru | Meta Refresh is an event on front-end engineering and design. |
 | [JSFoo](https://jsfoo.in/2016/) | 15-16 Sept | MLR Convention Centre, Bengaluru | JSFoo is India’s premier JavaScript conference. |
-| [SANOG](http://www.sanog.org/sanog28/) | 1-9 August | Trident, BKC, Mumbai | The South Asian Network Operator Group. Engineers sharing operational knowledge on running ISPs and modern networks. |
-| [The Fifth Elephant](https://fifthelephant.in/2016/) | 28-29 July | NIMHANS Convention Centre, Bengaluru | The Fifth Elephant is India’s most renowned conference on data, a space for discussing the most relevant tools, techniques and frameworks with expert practitioners in the field. |
-| [Rootconf](https://rootconf.in/2016/) | 14-15 Apr | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge about building resilient and scalable systems. |
-| [IndiaHacks Conference](https://www.hackerearth.com/indiahacks-conference/) | 19 Mar | Vivanta by Taj, Bengaluru | Conducted by HackerEarth,the conference has an extensive lineup of speakers talking on a variety of technical subjects like Machine Learning, Big Data, Artificial Intelligence, Scientific Computing, etc. |
+| [SANOG](http://www.sanog.org/sanog28/) | 1-9 Aug | Trident, BKC, Mumbai | The South Asian Network Operator Group. Engineers sharing operational knowledge on running ISPs and modern networks. |
+| [The Fifth Elephant](https://fifthelephant.in/2016/) | 28-29 July | NIMHANS Convention Centre, Bengaluru | The Fifth Elephant is India’s most renowned conference on data. |
+| [Rootconf](https://rootconf.in/2016/) | 14-15 Apr | MLR Convention Centre, Bengaluru | Rootconf is India’s principal conference where systems and operations engineers share real world knowledge. |
+| [IndiaHacks Conference](https://www.hackerearth.com/indiahacks-conference/) | 19 Mar | Vivanta by Taj, Bengaluru | Conference has an extensive lineup of speakers talking on a variety of technical subjects like ML, Big Data, AI etc. |
```

Comment:
Instead of `Conference has`, please make it `IndiaHacks has`. Btw, nice idea about shortening it to ML and AI.